IGC/AdaptorOCL/SPIRV/SPIRVReader.cpp

```diff
@@ -1925,7 +1929,11 @@ SPIRVToLLVM::transType(SPIRVType *T) {
         auto name = isSubgroupAvcINTELTypeOpCode(OC) ?
             OCLSubgroupINTELTypeOpCodeMap::rmap(OC) :
             BuiltinOpaqueGenericTypeOpCodeMap::rmap(OC);
+#if LLVM_VERSION_MAJOR >= 12
+        auto *pST = llvm::StructType::getTypeByName(M->getContext(), name);
+#else
         auto *pST = M->getTypeByName(name);
+#endif
```

Comment:
This should be wrapped in `LLVMWrapper` too.
LLVM change for future reference: https://reviews.llvm.org/D78793

Reply:
Done.